Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -89,7 +90,7 @@
         "maxPhiBins",
         "phi",  # (min,max)
     ],
-    defaults=[None] * 6 + [(None, None)] * 1,
+    defaults=[None] * 6 + [(-math.pi, math.pi)],
```

Comment:
Usually we use `None` here to default to the C++ options. Any reason to default it twice?

Reply:
right, this can indeed go back to `None`. It's a remnant of my testings

Reply:
done